XCUITests/NavigationTest.swift

```diff
@@ -303,15 +303,15 @@ class NavigationTest: BaseTestCase {
 
     func testShareLink() {
         longPressLinkOptions(optionSelected: "Share Link")
-        waitForExistence(app.collectionViews.cells["Copy"])
-        XCTAssertTrue(app.collectionViews.cells["Copy"].exists, "The share menu is not shown")
+        waitForExistence(app.collectionViews.buttons["Copy"])
```

Comment:
we may need a timeout there... looking again at the BB test failures, there are some related to this button:
https://dashboard.buddybuild.com/apps/57bf25c0f096bc01001e21e0/build/5f22e24575174000016a3d55/test/0902882b-282c-49c0-a748-afa6faf0348b/a6c1dcb4-5aef-4539-a619-20828c765956

let's try that timeout...also double check that is a button now instead of a cell...I saw that when reported the test status but I could be wrong and the element path is different 🤔